Remotion/ObjectBinding/UnitTests/BindableObject/BindableObjectBaseTest.cs

```diff
@@ -31,6 +34,14 @@ public class BindableObjectBaseTest
     private Mock<IBusinessObjectProperty> _propertyFake;
     private Mock<IBusinessObjectClass> _businessObjectClassFake;
 
+    [OneTimeSetUp]
+    public void OneTimeSetup ()
+    {
+      Assertion.IsTrue (
+          !typeof (BindableObjectMixin).CanAscribeTo (typeof (Mixin<,>)),
+          "we assume the mixin does not have a base object");
+    }
+
```

Comment:
```suggestion
    [Test]
    public void BindableObjectMixin_ImplementsMixinWithoutBaseObjectDependency ()
    {
      Assertion.That (typeof (BindableObjectMixin).CanAscribeTo (typeof (Mixin<,>)), Is.False);
    }
    
```
Please also move down to the test code.